solidity/test/FullyBackedECDSAKeepFactoryTest.js

```diff
@@ -0,0 +1,1633 @@
+        application
+      )
+
+      const res = await keepFactory.createSortitionPool(application)
+      truffleAssert.eventEmitted(res, "SortitionPoolCreated", {
+        application: application,
+        sortitionPool: sortitionPoolAddress,
+      })
+    })
+
+    it("reverts when sortition pool already exists", async () => {
+      await keepFactory.createSortitionPool(application)
+
+      await expectRevert(
+        keepFactory.createSortitionPool(application),
+        "Sortition pool already exists"
+      )
+    })
+  })
+
+  describe("getSortitionPool", async () => {
+    before(async () => {
+      await initializeNewFactory()
+    })
+
+    it("returns address of sortition pool", async () => {
+      const sortitionPoolAddress = await keepFactory.createSortitionPool.call(
+        application
+      )
+      await keepFactory.createSortitionPool(application)
+
+      const result = await keepFactory.getSortitionPool(application)
+      assert.equal(
+        result,
+        sortitionPoolAddress,
+        "incorrect sortition pool address"
+      )
+    })
+
+    it("reverts if sortition pool does not exist", async () => {
+      await expectRevert(
+        keepFactory.getSortitionPool(application),
+        "No pool found for the application"
+      )
+    })
+  })
+
+  describe("isOperatorRegistered", async () => {
+    before(async () => {
+      await initializeNewFactory()
+      await initializeMemberCandidates()
+    })
+
+    it("returns true if the operator is registered for the application", async () => {
+      await keepFactory.registerMemberCandidate(application, {
+        from: members[0],
+      })
+
+      assert.isTrue(
+        await keepFactory.isOperatorRegistered(members[0], application)
+      )
+    })
+
+    it("returns false if the operator is registered for another application", async () => {
+      const application2 = "0x0000000000000000000000000000000000000002"
+
+      await keepFactory.registerMemberCandidate(application, {
+        from: members[0],
+      })
+
+      assert.isFalse(
+        await keepFactory.isOperatorRegistered(members[0], application2)
+      )
+    })
+
+    it("returns false if the operator is not registered for any application", async () => {
+      assert.isFalse(
+        await keepFactory.isOperatorRegistered(members[0], application)
+      )
+    })
+  })
+
+  describe("isOperatorUpToDate", async () => {
+    const precision = new BN("1000000000000000000") // 1 ETH
+
+    let minimumBondableValue
+
+    before(async () => {
+      await initializeNewFactory()
+      await initializeMemberCandidates()
+
+      const pool = await FullyBackedSortitionPool.at(signerPool)
+      minimumBondableValue = await pool.getMinimumBondableValue()
+    })
+
+    // TODO: Add a case to check if operator is initialized
+    // github.com/keep-network/sortition-pools/blob/413abd66536032ee757f190a29fda2705c7b70cf/contracts/FullyBackedSortitionPool.sol#L155
```

Comment:
Should we?

Reply:
It was already covered in registration test: https://github.com/keep-network/keep-ecdsa/blob/d1edab304c4ace73f720f6c4e6d429a14ba6a19b/solidity/test/FullyBackedECDSAKeepFactoryTest.js#L221

It won't be possible to happen on `isOperatorUpToDate` as operator would be initialized already once they join the pool.

I'll remove this comment.